cicd/1-setup/cicd-dependencies.template.yml

```diff
@@ -141,6 +141,19 @@ Resources:
               - sts:AssumeRole
       ManagedPolicyArns:
         - 'arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy'
+  
+  ECSTaskRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - ecs-tasks.amazonaws.com
+            Action:
+              - sts:AssumeRole
       Policies:
```

Comment:
Is this for the future CICD work, or is this needed for the upload to s3 script?

Reply:
I'm tracking changes to this template and applying them manually to production as we iterate on them. The alternative would be to pop these out into their own PR and merge them ahead of this to get them working, that's just a little more work, and it's nice to see it all in this PR.